.gitignore

```diff
@@ -21,3 +21,4 @@ migrations/
 truffle-config.js
 test/config/ensAddresses.json
 */**/lib
+.env
```

Comment:
```suggestion
```

I think you can remove this since a user shouldn't create their `.env` within the web3 repo, but would instead create it in their specific project's directory